db/config.cc

```diff
@@ -798,13 +802,12 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "Sets the sending socket buffer size in bytes for remote procedure calls.")
     , rpc_server_type(this, "rpc_server_type", value_status::Unused, "sync",
         "Cassandra provides three options for the RPC server. On Windows, sync is about 30% slower than hsha. On Linux, sync and hsha performance is about the same, but hsha uses less memory.\n"
-        "\n"
-        "\tsync    (Default One thread per Thrift connection.) For a very large number of clients, memory is the limiting factor. On a 64-bit JVM, 180KB is the minimum stack size per thread and corresponds to your use of virtual memory. Physical memory may be limited depending on use of stack space.\n"
-        "\thsh      Half synchronous, half asynchronous. All Thrift clients are handled asynchronously using a small number of threads that does not vary with the number of clients and thus scales well to many clients. The RPC requests are synchronous (one thread per active request).\n"
-        "\t         Note: When selecting this option, you must change the default value (unlimited) of rpc_max_threads.\n"
-        "\tYour own RPC server: You must provide a fully-qualified class name of an o.a.c.t.TServerFactory that can create a server instance.")
+        "* sync    (Default One thread per Thrift connection.) For a very large number of clients, memory is the limiting factor. On a 64-bit JVM, 180KB is the minimum stack size per thread and corresponds to your use of virtual memory. Physical memory may be limited depending on use of stack space.\n"
+        "* hsh      Half synchronous, half asynchronous. All Thrift clients are handled asynchronously using a small number of threads that does not vary with the number of clients and thus scales well to many clients. The RPC requests are synchronous (one thread per active request).\n"
+        "*         Note: When selecting this option, you must change the default value (unlimited) of rpc_max_threads.\n"
+        "* Your own RPC server: You must provide a fully-qualified class name of an o.a.c.t.TServerFactory that can create a server instance.")
     , cache_hit_rate_read_balancing(this, "cache_hit_rate_read_balancing", value_status::Used, true,
-        "This boolean controls whether the replicas for read query will be chosen based on cache hit ratio")
+        "This boolean controls whether the replicas for read query will be choosen based on cache hit ratio.")
```

Comment:
@dgarcia360 could you please revert this change? i think "chosen" is correct. see https://www.merriam-webster.com/dictionary/choose

Reply:
Good catch! I'm not sure how the extra "o" got there.